app/assets/javascripts/discourse/templates/modal/invite.hbs

```diff
@@ -11,9 +11,13 @@
   {{else}}
 
     <label>{{inviteInstructions}}</label>
-    {{text-field value=email placeholderKey="topic.invite_reply.email_placeholder"}}
+    {{#if allowExistingMembers}}
```

Comment:
all this code feels very duplicate, we have it in invite into PM we need to use a single temple here to do both and a single controller